### ISSUE TITLE
TypeScript types can't be parsed properly after install

### ISSUE THREAD

**Issue by sjdemartini** [thomasbrodusch/vitest-fail-on-console#1]
Excited to come across this package, thanks for building it! (I was using the equivalent in Jest before, and had to remove it when switching to vitest).

---

I've installed this package and it seems to _run_ properly, but in my editor (VSCode) and when running `tsc --no-edmit`, I get errors for the line that imports this package:

```
ERROR(TypeScript)  Cannot find module 'vitest-fail-on-console' or its corresponding type declarations.
```

pointing to the line:

```ts
import failOnConsole from "vitest-fail-on-console";
```

I'm using yarn, TS 4.9, and have installed this package via `"vitest-fail-on-console": "^0.2.0",` in my package.json.

**Comment by thomasbrodusch** [comment]
Hi, @sjdemartini thank you for using `vitest-fail-on-console`, your precious feedback, and this first issue 🥳
After some investigations, I can tell you that I can reproduce that issue. 

The error thrown by  `tsc --noEmit` occurs due to a TypeScript type definition file name mismatch (it cannot find it).
That being said, I'll push a PR during the day to fix it.

**Comment by thomasbrodusch** [comment]
@sjdemartini fixed by 28a61de151e8b2aec74bbbd5ad0752e452b6dd97 
`v0.2.3` Deployed on NPM 🚀

**Comment by sjdemartini** [comment]
Perfect, it's working for me now! Thank you for the quick response and quick fix! 😄
